pkg/state/filter.go

```diff
@@ -102,3 +107,18 @@ func filterOut(currentState shared.State, interfacesFilterGlob glob.Glob) (share
 
 	return shared.NewState(string(filteredState)), nil
 }
+
+// normalizeInterfacesNames fixes the unmarshal of numeric values in the interfaces names
+// Numeric values, including the ones with a base prefix (e.g. 0x123) should be stringify.
+func normalizeInterfacesNames(rawState []byte, state *rootState) error {
+	var stateForNormalization rootState
+	if err := goyaml.Unmarshal(rawState, &stateForNormalization); err != nil {
+		return err
+	}
+	for i, iface := range stateForNormalization.Interfaces {
+		if isNumeric(iface.Name) {
+			state.Interfaces[i].Name = iface.Name
+		}
```

Comment:
I think checking `isNumeric` has the same cost as the Name copy, so we can just copy all the names and then we don't need `isNumeric`

Reply:
Yes, right. I had a problem with some double quoting, but with the latest version it is ok.
Done